gateway/chain.json

```diff
@@ -0,0 +1,128 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "gateway",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://wormhole.com",
+  "pretty_name": "Wormhole Gateway",
+  "chain_id": "wormchain",
+  "bech32_prefix": "wormhole",
```

Comment:
Please register this prefix: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
@JeremyParish69 see https://github.com/satoshilabs/slips/pull/1590